app/src/openxr/cpp/OpenXRInputMappings.h

```diff
@@ -116,6 +116,7 @@ namespace crow {
         std::vector<OpenXRInputProfile> profiles;
         std::vector<OpenXRButton> buttons;
         std::vector<OpenXRAxis> axes;
+        uint8_t numHaptics;
```

Comment:
Having an integer alone there is very confusing. I think we should have a structure similar to what we have for axis for example. We need at least the path and whether it applies to one hand or both (all details here https://registry.khronos.org/OpenXR/specs/1.0/html/xrspec.html#semantic-path-interaction-profiles) Note that most controllers only expose output/haptic but there are special cases that use other paths. We should not assume that output/haptic is the path for any controller.

Then with those structures we should initialize the number you were usong ensuring that we apply them for the proper actions for each hand

Reply:
Yes, that would be much more robust and future proof.

I have pushed a new version that tries to implement what you suggest above. Lets see :) 

Thanks for the feedback!